nzedb/Binaries.php

```diff
@@ -724,27 +748,33 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 					// Get the current unixtime from PHP.
 					$now = time();
 
+					$xref = ($multiGroup === true ? sprintf('xref = CONCAT(xref, "\\n"%s ),', $this->_pdo->escapeString(substr($header['Xref'], 2, 255))) : '');
```

Comment:
We actually never use xref value for anything up until recently. Now we use Xref to add xposted groups to nzb. As collections are deleted upon release creation, there is no need for "original" xref